tests/web/Glpi/Inventory/Request.php

```diff
@@ -40,7 +40,7 @@
 use GuzzleHttp\Exception\RequestException;
 use GuzzleHttp\Psr7\Response;
 
-class Request extends \GLPITestCase
+class Request extends \DBTestCase
```

Comment:
Is this change really required?

Reply:
Yes because we need to login before updating the config  (`login()` is available from `DBTestCase`)
because `saveConf` check for `Config::canUpdate()`
example : 
```php
 $this->login();
 $this->boolean($conf->saveConf(
            [
                "enabled_inventory" => true,
                'auth_required' => \Glpi\Inventory\Conf::BASIC_AUTH,
                'basic_auth_login' => $basic_auth_login,
                'basic_auth_password' => $basic_auth_password
            ]
        ))->isTrue();
```

Reply:
OK, but DBTestCase is meant to work from the same call, not from an external one - a transaction is created and never commited.
You have to explicitly commit all database change so they are persisted - and you also have to reset properly previous state and commit explicitly too. I'm not sure what else exactly it implies (just check beforeTestsMethod and other in the DbTestCase class.